tools/test_containers_compose

```diff
@@ -1,12 +1,22 @@
 #!/bin/bash
 
 set -euo pipefail
+#https://github.com/os-autoinst/openQA/pull/3864/checks?check_run_id=2440471511
+#https://github.com/os-autoinst/openQA/pull/3864/checks?check_run_id=2444792522
```

Comment:
I assume these are just temporary notes by you?

Reply:
Yes this is under draft and WIP